services/sso/src/controllers/your-planning-appeal.js

```diff
@@ -0,0 +1,22 @@
+const { VIEW } = require("../lib/views");
```

Comment:
As this is a common service it should ideally be more generic and not include controllers for specific services as it shouldn't need to know about the services that consume it. Does this need to be included here?